README.md

```diff
@@ -106,19 +109,19 @@ The map that will be loaded for the upcoming round is determined by reading data
 
 If you are hosting a server, and want randomly picked maps to be played each round, you can enable map rotation in [config.txt](config/config.txt) and then set the maps to be picked in the [maps.txt](config/maps.txt) file.
 
-Anytime you want to make changes to a map it's imperative you use the [Map Merging tools](https://tgstation13.org/wiki/Map_Merger)
+Anytime you want to make changes to a map it's imperative you use the [Map Merging tools](https://wiki.yogstation.net/wiki/Map_Merger)

```

Comment:
Keep this because /tg/ wiki is more up2date than ours

Reply:
Seems like a wiki problem but I'll fix it until ours are up to date then.